scripts/libff.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-set -ev
+set -e
```

Comment:
We have code like this below, but with `set -e`, it'll exit as soon as there's a fail, not even printing the message. Isn't that bad?

```cmake ../src -DCMAKE_INSTALL_PREFIX=../install -DCMAKE_POSITION_INDEPENDENT_CODE=1 -DWITH_PROCPS=OFF
if [[ $? -ne 0 ]]
then
    echo "libff: CMake configuration failed"
    exit 1
fi
```

Reply:
I remember seeing something in this script failing but still trying to execute commands. And that didn't help with opam installation. Can revert it though.